### ISSUE TITLE
[a11y] Usage of colours 

### ISSUE THREAD

**Issue by chmst** [lupoair/caneveryoneuse#41]
For colour blind the usage of colours as only information is not accessible. https://www.w3.org/WAI/WCAG21/Techniques/general/G14
found here: https://tool.caneveryoneuse.com/framework-overview
